manage_cortex_prs.sh

```
@@ -0,0 +1,435 @@
+
+set -e
+
+echo "🧠 CORTEX LINUX - MASTER PR CONTROL SYSTEM"
+echo "=========================================="
+echo ""
+
+# Configuration
+REPO="cortexlinux/cortex"
+REPO_DIR="$HOME/cortex"
+DISCORD_INVITE="https://discord.gg/uCqHvxjU83"
+GITHUB_TOKEN=$(grep GITHUB_TOKEN ~/.zshrc | cut -d'=' -f2 | tr -d '"' | tr -d "'")
```

Shellcheck (0.11.0)

[warning] GITHUB_TOKEN appears unused. Verify use (or export if used externally).

(SC2034)